themes/src/main/resources/theme/keycloak.v2/account/src/package.json

```diff
@@ -25,6 +23,7 @@
     "@patternfly/react-core": "^4.147.0",
     "@patternfly/react-icons": "^4.11.8",
     "@patternfly/react-styles": "^4.11.8",
+    "keycloak-js": "workspace:*",
```

Comment:
Since the Account Console v2 is now in the same workspace, we can get the `keycloak-js` dependency straight from there.